build.gradle

```diff
@@ -127,6 +132,6 @@ task deployNodes(type: net.corda.plugins.Cordform, dependsOn: ['jar']) {
             address("localhost:10006")
             adminAddress("localhost:10046")
         }
-        rpcUsers = [[ user: "beycoadmin", "password": "beycoadmin", "permissions": ["ALL"]]]
+        rpcUsers = [[ user: "test_admin_beyco_user", "password": "test_admin_beyco_pass", "permissions": ["ALL"]]]
```

Comment:
Wachtwoorden wil je eigenlijk nooit in de code mee committen. Ik weet niet of java hier een oplossing voor heeft? In .NET hebben we een 'appsettings.json' bestand dat we inladen en de wachtwoorden en belangrijke settings uit halen.